components/renku_data_services/project/api.spec.yaml

```diff
@@ -148,6 +148,59 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - projects
+  /renku_v1_projects/{v1_id}/migrations:
+    get:
+      summary: Check if a v1 project has been migrated to v2
+      parameters:
+        - in: path
+          name: v1_id
+          required: true
+          description: The ID of the project in Renku v1
+          schema:
+            type: integer
+      responses:
+        "200":
+          description: Project exists in v2 and has been migrated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Project"
+        "404":
+          description: No corresponding project found in v2
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - projects
+    post:
+      summary: Create a new project migrated from Renku v1
```

Comment:
suggestion: document that this will not just create the project, but also session launchers etc. otherwise it seems like we just have two endpoints that create projects.

I'm also a bit torn on having this be a separate endpoint instead of just calling the regular project create, session launcher add endpoints and then here just storing the v1 id on the created project. that way we wouldn't have to duplicate all the session schemas in the project API.

Reply:
@Panaetius I was thinking we should just combine all the apispec.yaml files for the whole code base into one place. Then duplicating the schemas in the project API (or other places) is less of a concern. But combining the apispec.yaml files is a separate  task - I am not proposing to do this here. Just saying that we can address this in a slightly different way for all the components.